openquake/commonlib/tests/logictree_test.py

```diff
@@ -1468,8 +1468,7 @@ def test_not_a_gsim_logic_tree(self):
             </logicTree>
         """)
         self.parse_invalid(
-            xml, logictree.InvalidLogicTree, "Could not find branches with "
-            "attribute 'applyToTectonicRegionType' in set(['Shield'])")
+            xml, NameError, 'Unknown GSIM: +100 in file <StringIO>')
```

Comment:
It is a test for invalid logic tree. In particular here we are considering the case when the user has used a source logic tree instead of a gmpe logic tree, exactly this:

```
            <logicTree logicTreeID="lt1">
              <logicTreeBranchingLevel branchingLevelID="bl1">
                <logicTreeBranchSet uncertaintyType="bGRRelative"
                                    branchSetID="bs1">
                  <logicTreeBranch branchID="b1">
                    <uncertaintyModel>+100</uncertaintyModel>
                    <uncertaintyWeight>1.0</uncertaintyWeight>
                  </logicTreeBranch>
                </logicTreeBranchSet>
              </logicTreeBranchingLevel>
            </logicTree>
```